layouts/shortcodes/redhat/latest-version.html

```diff
@@ -1 +1 @@
-1.6.0
\ No newline at end of file
+1.9
```

Comment:
The new system (https://github.com/kubeflow/website/pull/3842) requires you to specify a specific version patch (it removes the patch version from the outer table but shows the full version inside the pop-up).

Please make this 1.9.0.